tests/testosx.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# This file is part of NIT ( http://www.nitlanguage.org ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Tests to run on OS X for the host platform and iOS
+./tests.sh $@ \
+	test_ffi_objc_*.nit \
+	test_*ios.nit \
+	../lib/ios/examples/*.nit \
+	../lib/cocoa*/examples/*.nit
```

Comment:
I did completely forget there already is a `$uname.skip` infrastructure. You could then easily add a`Darwin.skip` to have 100% on http://gresil.org/jenkins/job/CI%20Github%20OSX/